lib/cargoTarpaulin.nix

```diff
@@ -3,7 +3,7 @@
 }:
 
 { cargoExtraArgs ? ""
-, cargoTarpaulinExtraArgs ? "--skip-clean --out Xml --output-dir $out"
+, cargoTarpaulinExtraArgs ? "--skip-clean --out xml --output-dir $out"
```

Comment:
The 23.05 branch currently has cargo-tarpaulin `0.25.2` which accepts the `Xml` format, while the version in nixpkgs-unstable is at `0.27.0` so something must have changed there...

We could introspect the cargo-tarpaulin version and set the default here accordingly (at least until the 23.11 is released)

(Also docs are great as is, we don't need to highlight this handling there)

```nix
let
  xml = if lib.versionOlder cargo-tarpaulin.version "0.27.0" then "Xml" else "xml";
in
{ cargoExtraArgs ? ""
, cargoTarpaulinExtraArgs ? "--skip-clean --out ${xml} --output-dir $out"
, ...
}:
```

Reply:
Thanks for taking a look, I've added the version check.